Sprintfinity3902/Entities/Link/Player.cs

```diff
@@ -77,10 +87,23 @@ public Player()
             linkHealth = MAX_HEALTH;
             heartChanged = true;
             itemPickedUp = false;
+            lowHealthInstanceID = SoundManager.Instance.RegisterSoundEffectInst(SoundLoader.Instance.GetSound(SoundLoader.Sounds.LOZ_LowHealth), 0.02f, true);
+            _deathSpinCount = 0.0;
 
             itemcount = new Dictionary<IItem.ITEMS, int>();
         }
 
+        public void Initialize() {
+            KeyboardManager.Instance.RegisterCommand(new SetPlayerMoveUpCommand(game.link), Keys.W, Keys.Up);
+            KeyboardManager.Instance.RegisterCommand(new SetPlayerMoveLeftCommand(game.link), Keys.A, Keys.Left);
+            KeyboardManager.Instance.RegisterCommand(new SetPlayerMoveDownCommand(game.link), Keys.S, Keys.Down);
+            KeyboardManager.Instance.RegisterCommand(new SetPlayerMoveRightCommand(game.link), Keys.D, Keys.Right);
+
+            KeyboardManager.Instance.RegisterKeyUpCallback(() => {
+                CurrentState.Sprite.Animation.Stop();
+            }, Keys.W, Keys.A, Keys.S, Keys.D, Keys.Up, Keys.Down, Keys.Left, Keys.Right);
+        }
```

Comment:
What value does moving this into player give us? I'm not inherently opposed, but i did want to keep keyboard commands outside of player.

Reply:
Um... I'm pretty sure it **doesn't help us at all this way**. We have opposing preferences; I'm biased to have it this way because it sounds like Game1 is expected to be "short" which I think is dumb but was said during class. Also, it makes more sense to me that the player would add their own commands. All this said, I really don't care **at all** where these commands are, my brain just defaults to pulling things out of Game1. If you want me to make another commit with these back in Game1 I can do that or otherwise you're more than welcome to pull these back out to Game1 in a future commit. Whatever the case, a quick comment will definitely remind me not to move their parent class anywhere.

Reply:
No need for a new commit! This is good for now, I would like to find it a better home in the future though.